lib/install/bin/webpack-dev-server.tt

```diff
@@ -16,13 +16,19 @@ CONFIG_FILE       = File.join(APP_PATH, "config/webpacker.yml")
 NODE_MODULES_PATH = File.join(APP_PATH, "node_modules")
 WEBPACK_CONFIG    = File.join(APP_PATH, "config/webpack/#{NODE_ENV}.js")
 
-LISTEN_HOST_ADDR = NODE_ENV == 'development' ? 'localhost' : '0.0.0.0'
-
 def args(key)
   index = ARGV.index(key)
   index ? ARGV[index + 1] : nil
 end
 
+LISTEN_HOST_ADDR = if args('--listen-host')
```

Comment:
I think this could be cleaner with something like this...

First, just rename the current LISTEN_HOST_ADDR to something like DEFAULT_LISTEN_HOST_ADDR:

```ruby
DEFAULT_LISTEN_HOST_ADDR = NODE_ENV == 'development' ? 'localhost' : '0.0.0.0'
```

Then within the begin and parallel to the other constants like (DEV_SERVER_ADDR) do this:

```ruby
  HOSTNAME        = args('--host') || dev_server["host"]
  PORT            = args('--port') || dev_server["port"]
  HTTPS           = ARGV.include?('--https') || dev_server["https"]
  DEV_SERVER_ADDR = "http#{"s" if HTTPS}://#{HOSTNAME}:#{PORT}"
  LISTEN_HOST_ADDR = args('--listen-host') || DEFAULT_LISTEN_HOST_ADDR
```

Reply:
thanks, pushed an update 👍  💯